This patch is made by paissad <paissad@gmail.com>
We only changed the dir when the log files are saved by default,
we changed to /var/log/pms-linux instead of /tmp
Thus, the log files are easily rotated (logrotate).
--- PMS.java.orig	2010-03-01 22:22:20.000000000 +0100
+++ PMS.java	2010-03-16 01:47:43.000000000 +0100
@@ -282,17 +282,21 @@
 		
 		registry = new WinUtils();
 					
+		// Beginning of changes Slackware package maintainer
+		String username;
+		username = System.getProperty("user.name");
 		File debug = null;
 		try {
-			debug = new File("debug.log"); //$NON-NLS-1$
+			debug = new File("/var/log/pms-linux/debug-"+username+".log"); //$NON-NLS-1$
 			pw = new PrintWriter(new FileWriter(debug)); //$NON-NLS-1$
 		} catch (Throwable e) {
-			PMS.minimal("Error in accessing debug.log..."); //$NON-NLS-1$
+			PMS.minimal("Error in accessing debug-"+username+".log..."); //$NON-NLS-1$
 			pw = null;
 		} finally {
 			if (pw == null) {
-				PMS.minimal("Using temp folder for debug.log..."); //$NON-NLS-1$
+				PMS.minimal("Using temp folder for debug-"+username+".log..."); //$NON-NLS-1$
 				debug = new File(configuration.getTempFolder(), "debug.log"); //$NON-NLS-1$
+		// end modification
 				pw = new PrintWriter(new FileWriter(debug)); //$NON-NLS-1$
 			}
 		}
